README.md

```diff
@@ -1,2 +1,58 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # capstone-1
 Building an online website for a conference
+
+![screenshot](./images/2021-10-12.png)
+
+.
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+## Application Page url
+
+https://phelian23.github.io/capstone-1/
+
```

Comment:
- [ ] It would be better if the live demo link shows the application. Currently, the link is showing me the readme file. 

![Capture](https://user-images.githubusercontent.com/64173189/137174088-bf196e30-5616-4cf7-8c74-a2690082a400.JPG)

Please check the this minor issue 

😄 🛠️